bundles/org.openhab.ui/web/src/pages/settings/persistence/persistence-edit.vue

```diff
@@ -162,9 +162,12 @@
               </div>
             </div>
           </f7-col>
-          <f7-col v-if="isEditable && !newPersistence">
+          <f7-col>
             <f7-list>
-              <f7-list-button color="red" @click="deletePersistence">
+              <f7-list-button color="blue" :href="`https://${$store.state.runtimeInfo.buildString === 'Release Build' ? 'www' : 'next'}.openhab.org/docs/configuration/persistence.html`" external target="_blank">
+                Open documentation
+              </f7-list-button>
```

Comment:
A list button is not the best for linking to the docs IMHO, those are better suited for internal actions. For those concepts that are a little obscure and need a link to the docs a f7-link labeled "Learn more [about X]" or "About X" would be better, see the profile configuration for example (the blurb before the link can be optional):

https://github.com/openhab/openhab-webui/blob/9be42930d957c22e55c7b5e7931a9f24123e2c82/bundles/org.openhab.ui/web/src/pages/settings/things/link/link-edit.vue#L49-L53

or iOS settings (note how the list buttons does something while the documentation links are simply blue hyperlinks):

![image](https://github.com/openhab/openhab-webui/assets/2004147/7488f8c8-2aa2-464f-a298-55c136107858)

Also, as shown above for profiles (the www/next switch is missing there, sadly!) it would be better to create a `openhab.org/link/persistence` alias link or similar, that way if the docs are reorganized it wouldn't require a change in the code but only an update to the redirect. This happens here: https://github.com/openhab/website/blob/main/.vuepress/_redirects

Reply:
I've added a short paragraph with the link to the top and created a PR for the redirect, thanks for your feedback, looks much better now!

See https://github.com/openhab/website/pull/413.